portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,16 +36,19 @@
 public class DataServlet extends HttpServlet {
   private final String COMMENT = "comment";
   private final String NAME = "name";
-  
+  private final String MAX_COMMENTS_LOAD = "loadcommentquantity";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("comment").addSort("timestamp", SortDirection.DESCENDING);
+    int maxCommentsLoad = Integer.parseInt(request.getParameter(MAX_COMMENTS_LOAD));
```

Comment:
We probably need to deal with the error case where the string to parse is not a valid integer, e.g., client side sends a wrong parameter. Here, specifically, parseInt may throw an exception and you need to catch and handle that.

Reply:
Done